api-docs/schemas/dataSetMetadata.yaml

```diff
@@ -0,0 +1,26 @@
+type: object
```

Comment:
I think we should add example blocks for all of our schemas. These will make more sense in the docs than filling the values in as the types.

Reply:
Ahh just found the docs on how to add examples. I'll do that for all the schemas